digdag-ui/console.jsx

```diff
@@ -1326,7 +1372,7 @@ class TaskListView extends React.Component {
           </thead>
           <tbody>
             {
-              Array.from(this.props.tasks.values()).map(task =>
+              sortTasksForTreeView(Array.from(this.props.tasks.values())).map(task =>
```

Comment:
I'm not 100% sure but, I think `this.props.tasks.values()` already returns an array, why casting it to array again?